examples/tutorials/Part 12 bis - Encrypted Training on MNIST.ipynb

```diff
@@ -0,0 +1,664 @@
+{
```

Comment:
<p>Not sure if this python snippet is formatted correctly (it doesn't seems so at app.reviewnb.com)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2411/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Looks really nice on jupyter notebook (I used `x3 and "python" keyword)</p>